src/scripts/detector/qr_detector.py

```diff
@@ -91,7 +91,7 @@ def detect(self, img):
         detections_res = qr_detector.detect(img)
         observations_mapobj = map(
             lambda qr_code: observation(
-                self.name, 0, str(qr_code['data']), 1,
+                self.name, (1 << 16)-1, str(qr_code['data']), 1,
```

Comment:
Onko joku syy tehdä tää bit shiftillä ihan vaan 65535 sijaan? Että sen näkee helpommin olevan 2^16 - 1?

Reply:
Juurikin näin.